src/MerchantCenter/MerchantVerification.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Merchant;
+use Automattic\WooCommerce\GoogleListingsAndAds\Infrastructure\Service;
+use Exception;
+use Google\Service\ShoppingContent\Account;
+use Google\Service\ShoppingContent\AccountBusinessInformation;
+
+/**
+ * Class MerchantVerification.
+ *
+ * @since x.x.x
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter
+ */
+class MerchantVerification implements Service {
+
+	/**
+	 * @var Merchant
+	 */
+	protected $merchant;
+
+	/**
+	 * MerchantVerification constructor.
+	 *
+	 * @param Merchant $merchant
+	 */
+	public function __construct( Merchant $merchant ) {
+		$this->merchant = $merchant;
+	}
+
+	/**
+	 * Get the phone number for the connected Merchant Center account.
+	 *
+	 * @return string The phone number associated with the Merchant Center account.
+	 * @throws Exception If the Merchant Center account can't be retrieved.
```

Comment:
Currently, the `Merchant` class catches the `GoogleException` and throws a generic `Exception`, but would this be what you have in mind: f63e72deb9e4d12d11c475babe840de2f14bc96e ?

Reply:
Yes 💯 that's pretty much what I had in mind. Although it'd be good to rename it to something more specific because `ContentAPIException` covers a lot of areas including product syncing, etc.

Reply:
Good point, had the blinders on and was only thinking about the Merchant usage of the Content API .